modules/search/instant-search/lib/query-string.js

```diff
@@ -188,3 +194,14 @@ export function setFilterQuery( filterKey, filterValue ) {
 	query[ filterKey ] = filterValue;
 	pushQueryString( encode( query ) );
 }
+
+export function getResultFormatQuery() {
+	const knownResultFormats = [ RESULT_FORMAT_MINIMAL, RESULT_FORMAT_PRODUCT ];
```

Comment:
We could move this outside the function to avoid instantiating it on every invocation.